remote/remote.go

```diff
@@ -587,15 +597,16 @@ func attachWpCliCmdRemote(conn net.Conn, wpcli *wpCLIProcess, GUID string, rows
 				break Catchup_Loop
 			}
 
-			written, err = conn.Write(buf[:read])
+
+			_, err = connWriteUTF8(conn, buf[:read])
 			if nil != err {
 				log.Printf("attachWpCliCmdRemote catchup: error writing to client connection: %s\n", err.Error())
 				readFile.Close()
 				return
 			}
 
 			wpcli.padlock.Lock()
-			wpcli.BytesStreamed[remoteAddress] = wpcli.BytesStreamed[remoteAddress] + int64(written)
+			wpcli.BytesStreamed[remoteAddress] = wpcli.BytesStreamed[remoteAddress] + int64(len(buf[:read]))
```

Comment:
@luismulinari why the switch from `written` to `_`? 
Seems to me the `connWriteUTF8` should return similar elements, pointing that out mostly because it'll keep the code intact in these lines and the `written` var is more readable to me.
(not a big deal though ;) )

Reply:
Hey Andrea! Thanks for the review!

Great question!

`wpcli.BytesStreamed` controls how much data we already streamed. It is used to keep waiting until we stream all the data we've received.

Using `len(buf[:read])` I make sure that doesn't matter what kind of changes we did on the bytes (like `bytes.ToValidUTF8`) we are actually incrementing the "same amount of data" to the client. I think it will help to keep consistency between how much data we received/sent. 

Does it make sense?

Reply:
Make sense. 
Do you think it would be possible to return that length directly from the `connWriteUTF8`? 
It'll help avoid thinking too much and remembering about this logic change.

Otherwise we would need to keep track of when we want the original length or another. This way we could chose.
If that's not feasable then it's fine like it is now ;) it's just to make things easier for new developers